### ISSUE TITLE
Errors thrown in a vm context do not have their stack printed

### ISSUE THREAD

**Issue by pmalouin** [nomiddlename/log4js-node#252]
I run some code inside a node.js vm:

```
var log4js = require('log4js');
var vm = require('vm');

var logger = log4js.getLogger();

try {
    // Access not defined variable.
    vm.runInNewContext('myVar();', {}, 'myfile.js');
} catch (e) {
    // Expect to have a stack trace printed.
    logger.error(e);
}
```

The log printed in the console does not have a stack trace printed:
[2014-10-16 17:23:27.483] [ERROR] [default] - [ReferenceError: myVar is not defined]

The reason I came up with is that in the `wrapErrorsWithInspect` function of lib/layouts.js, we check for `item instanceof Error`, but it is evaluated to false because Errors thrown from a new context refer to a different Error type instance. (Some explanations are given here: https://github.com/joyent/node/issues/1277)

I tested a different way to check for errors:

```
util.isError(item)
```

And it works in my case.

**Comment by nomiddlename** [comment]
This seems like such an easy fix - sorry for having ignored this for 3 years (I was obviously very busy in 2014).

**Comment by nomiddlename** [comment]
published to npm in version 2.3.1
